spec/formatting.md

```diff
@@ -219,15 +224,68 @@ the following steps are taken:
    as long as reasonable precautions are taken to keep the function interface
    simple and minimal, and avoid introducing potential security vulnerabilities.
 
-   As implementations MAY allow custom functions to be defined by users,
-   their access to the _formatting context_ SHOULD be minimal and read-only,
-   and their execution time SHOULD be limited.
+   Implementations MAY expose _options_ in an implementation-defined _namespace_
+   in addition to or superseding those found in the default registry.
```

Comment:
I don't understand how including or not including this sentence changes what implementations and users are able or allowed to do.

Reply:
Rmoeved